bundles/AdminBundle/GDPR/DataProvider/Exporter.php

```diff
@@ -0,0 +1,179 @@
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\AdminBundle\GDPR\DataProvider;
+
+use Pimcore\Logger;
+use Pimcore\Model\Asset;
+use Pimcore\Model\DataObject\AbstractObject;
+use Pimcore\Model\DataObject\ClassDefinition\Data;
+use Pimcore\Model\DataObject\Concrete;
+use Pimcore\Model\DataObject\Fieldcollection;
+use Pimcore\Model\DataObject\Objectbrick;
+
+/**
+ * Class Exporter
+ * @package Pimcore\Bundle\AdminBundle\GDPR\DataProvider
+ *
+ * @internal
+ * @final
+ */
+class Exporter
```

Comment:
we should make this independent service. please see https://github.com/pimcore/advanced-object-search/issues/90

Reply:
IMHO, it should be decoupled from the core, why again provide a centralized service ? wdyt ?

Reply:
In this particular case I would also think it make sense to keep it decoupled.